internal/kojiapi/server.go

```diff
@@ -384,6 +384,43 @@ func (h *apiHandlers) GetComposeIdLogs(ctx echo.Context, idstr string) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+// GetComposeIdManifest returns the Manifest for a given Compose.
+func (h *apiHandlers) GetComposeIdManifest(ctx echo.Context, idstr string) error {
+	id, err := uuid.Parse(idstr)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Invalid format for parameter id: %s", err))
+	}
+
+	var finalizeResult worker.KojiFinalizeJobResult
+	_, deps, err := h.server.workers.JobStatus(id, &finalizeResult)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Job %s not found: %s", idstr, err))
+	}
+
+	if len(deps) > 1 {
+		// Get the first build job ID (second dependency) to get its manifest
```

Comment:
Hmm, different architectures have different manifests so we probably need to return all of them.